pb_deploy.sh

```diff
@@ -160,12 +159,12 @@ function gh_deploy() {
 function tg_official_deploy() {
 	echo -e "${green}Deploying to Telegram!\n${nocol}"
 
-	FORMAT="PitchBlack Recovery for <b>$TARGET_DEVICE</b> (<code>${NAME}</code>)\n\n<b>Info</b>\n\nPitchBlack V${VERSION} ${DEPLOY_TYPE_NAME}\nBased on TWRP ${TWRP_V}\n<b>Build Date</b>: <code>${BUILD_DATE:0:4}/${BUILD_DATE:4:2}/${BUILD_DATE:6}</code>\n\n<b>Maintainer</b>: ${maintainer}\n"
+	FORMAT="PitchBlack Recovery for <b>$TARGET_DEVICE</b> (<code>${CODENAME}</code>)\n\n<b>Info</b>\n\nPitchBlack V${VERSION} ${DEPLOY_TYPE_NAME}\nBased on TWRP ${TWRP_V}\n<b>Build Date</b>: <code>${BUILD_DATE:0:4}/${BUILD_DATE:4:2}/${BUILD_DATE:6}</code>\n\n<b>Maintainer</b>: ${maintainer}\n"
```

Comment:
Here can we change `TARGET_DEVICE` to actually get full device name stored in `pb_devices.json` ?

Reply:
hmmmm, lets see